src/app/components/policy-guide/docs/capacity/capacity-introduction/capacity-introduction.template.html

```diff
@@ -1,14 +1,15 @@
-<p><code>public affairs staff, open government staff, web manager or digital strategist, program owners,</code></p>
-<p>In order to be successful in meeting the requirements of the policy and realize real, sustained value from its investment in open source, your agency shouldn’t silo it as merely a technical or acquisitions issue.  Instead, the most successful agencies will approach open source in a similar manner to security and change management - as an essentially interdisciplinary capability that requires deliberate, formal, horizontal coordination across the organization.</p>
-<p>The most successful agencies will invest up front in creating consistent boilerplate and establishing clear policy and procedures that are shared across verticals. They will establish formal coordination bodies and integrate open source and source code management into existing coordinating functions in a formal way - again, security and change management are good examples.</p>
-<p>Further, agencies will benefit from approaching open source and source code management as capabilities that “almost already” exist. Required agency capabilities like asset management, security monitoring, and privacy represent elements of a successful open source program that already exist in your agency and are integrated into its day to day operations. Leveraging these teams and processes will likely result in a stronger and more cost efficient program.</p>
-<h1><a id="Consult_with_peer_agencies_and_share_practices_29"></a>Consult with peer agencies and share practices</h1>
-<p>In the spirit of open source development, agencies can benefit greatly from “forking” eachother’s approaches to open source generally and even specific language that has been developed with regards to issues like licensing and contract boilerplate. Complex topics like security, communications, and even how to run a successful hackathon are ripe for collaboration between agencies to share lessons learned and pool limited resources.</p>
+<h1>Building Your Agency's Open Source Practice</h1>
+<p><a href="https://policy.cio.gov/source-code/implementation/">Section 7.1</a> of the Source Code Policy discusses roles and responsibilities within agencies in meeting the overall requirements of the policy. Regarding the Open Source Pilot Program, it also notes that &ldquo;<em>agencies should strengthen internal capacity to efficiently and securely deliver OSS as part of regular operations.&rdquo; </em></p>
+<p>Whether it is with a focus on building capacity in software acquisition, development, or both, agencies will benefit from developing staff expertise in a number of areas. In some cases, agencies may also want to build technical capacity.</p>
+<p>The open source practice for the U.S. government is expected to rapidly evolve and become more sophisticated over the next several years, so the advice provided here will also evolve.</p>
+<p><strong>Engaged and supportive agency leadership</strong></p>
```

Comment:
could you not use an <h2> or something similar here and below? That would seem to be more accessibile-friendly since it looks like these are actually subheads